certified-connectors/Telstra SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,720 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Telstra SMS",
+    "description": "Send SMS \u0026 MMS, leveraging Australia\u0027s leading mobile network.",
+    "version": "1.0",
+    "termsOfService": "https://www.telstra.com.au/customer-terms",
+    "contact": {
+      "name": "Telstra Messaging API",
+      "url": "https://dev.telstra.com/apis/messaging-api",
+      "email": "telstradev@team.telstra.com"
+    }
+  },
+  "host": "products.api.telstra.com",
+  "basePath": "/messaging/v3",
+  "schemes": ["https"],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/messages": {
+      "post": {
+        "responses": {
+          "400": {
+            "description": "default",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc. but not default